src/Core/Domain/Order/QueryResult/OrderInvoiceAddressForViewing.php

```diff
@@ -136,6 +143,7 @@ public function __construct(
         $this->postCode = $postCode;
         $this->phoneNumber = $phone;
         $this->mobilePhoneNumber = $phoneMobile;
+        $this->dni = $dni;
```

Comment:
What about doing this? That way getDni could return a string
```suggestion
        $this->dni = (string) $dni;
```